src/util/dataHelpers.js

```diff
@@ -50,10 +50,10 @@ export function dataItem(data, item, valueField){
   // make an attempt to see if we were passed in dataItem vs just a valueField value
   // either an object with the right prop, or a primitive
   // { valueField: 5 } || "hello" [ "hello" ]
-  if (has(item, valueField) || typeof first === typeof val)
```

Comment:
Just realized this slipped in to the commit. A variable named 'val' does not seem to exist, so I took a guess that it was supposed to be 'item'. I couldn't tell that it made a difference either way.